crossplane.yaml

```diff
@@ -32,64 +32,11 @@ metadata:
 spec:
   crossplane:
-    version: ">=v1.2.0-0"
+    version: ">=v1.9.0-0"
```

Comment:
I think this says the minimum version of Crossplane that this package will install on.  Does this package require functionality in crossplane v1.9+? If not, then I think we shouldn't move past the currently supported versions of Crossplane, which still include v1.7.